metrics/prometheus.go

```diff
@@ -1907,6 +1907,18 @@ func (c *PrometheusCollector) collectContainersInfo(ch chan<- prometheus.Metric)
 				}
 			}
 		}
+		if c.includedMetrics.Has(container.PerfMetrics) {
+			perfErrors = append(perfErrors, cont.Stats[0].Perf.PerfErrors...)
+		}
+	}
+	if c.includedMetrics.Has(container.PerfMetrics) {
```

Comment:
I'm not sure if it a good place for creating this metrics, take a look what I made for machine metrics: https://github.com/google/cadvisor/blob/master/metrics/prometheus_machine.go#L129

Reply:
As far as I know this is only place for creating that metric. In prometheus_machine.go we don't have access to those structs. Metrics are now unique. It reports unique entries without any filtering from our side, so I suppose, that prometheus filters it out.